exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_log_record_exporter_factory.h

```diff
@@ -17,7 +17,7 @@ namespace otlp
 /**
  * Factory class for OtlpGrpcLogRecordExporter.
  */
-class OtlpGrpcLogRecordExporterFactory
+class OPENTELEMETRY_EXPORT OtlpGrpcLogRecordExporterFactory
```

Comment:
Should we also export factory of metric exporters?

Reply:
Yes, metrics will be in later PR.